microsite/src/main/tut/usage/schedules.md

```diff
@@ -21,7 +21,7 @@ Schedules allow you to define and compose flexible recurrence schedules, which c
    * `IO.retryOrElse` — Retries an action until it succeeds, with a fallback for errors.
    * `IO.retryOrElse0` — Retries an action until it succeeds, with a more powerful fallback for errors.
 
-Schedules define stateful, possibly effectful, recurring schedules of events, and compose in a variety of ways.
```

Comment:
I think we can leave the Oxford comma :)

Reply:
Secret is out. I don't know how to use it. I have reading to do

Reply:
fixed :)